### PR TITLE
Feat/dashboard

### LINKED ISSUE #591: Dashboard of history of tokens for lifebank user
Reference: #555 and #558

This dashboard will be available for a lifebank user so they can see a history of the movement of life tokens, either when they send or receive them. It must show:
- Date
- If they were sent or received
- Business where they were redeemed
- Offer redeemed

- [x] Backend #705
- [x] Front-end #706

### LINKED ISSUE #706: Front-end of tokens history dashboard for lifebank user
Dep: #591

Visualization of history dashboard ("ver historial"). A user can see a history of transactions and filter using dates (from - to). Using the same data that we use to export PDFs.

### PR DESCRIPTION
**Pull request by steph222**
Resolves #591  

### What does this PR do?
Show History Dashboard 

### How should this be tested?
- Make run
- Step 2
- Step 3

### CODE REVIEW

**Review by Edgar097 — APPROVED** [review]

File: webapp/src/language/es.json [hunk 1]
```diff
@@ -61,7 +61,8 @@
       "aboutLifebank": "Acerca de Lifebank",
       "useTerms": "Términos de uso",
       "help": "Ayuda",
-      "offersApproval": "Aprobar ofertas"
+      "offersApproval": "Aprobar ofertas",
+      "historyDashboard": "Ver historial"
```
[the remark this answers is not in the document]
Comment (steph222) [reply webapp/src/language/es.json:65]: It was the texts proposed by Luis Diego, it is not a translation

File: webapp/src/routes/index.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
 import NotFound from './NotFound'
 import InternalError from './InternalError'
 import Profile from './Profile'
+
```
Comment (leisterfrancisco) [inline webapp/src/routes/index.js:4]: remove empty line

Comment (leisterfrancisco) [reply webapp/src/routes/index.js:4]: I think you removed something that is needed.

Take a look at this compiling error:
Screen Shot 2021-06-14 at 08 37 05

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

File: webapp/src/routes/index.js (hunk 2 above)
Comment (steph222) [reply webapp/src/routes/index.js:4]: Don't delete it, the merge only brought that without the import

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]
Donor has received 8 tokens from lifebank but when I go to History Dashboard the content is empty:
Screen Shot 2021-06-15 at 08 12 30

**Comment by leisterfrancisco** [comment]
@steph222 please also merge master changes to keep to branch up to date

**Pull request merged**